EasyPost.Tests/ServicesTests/Beta/RateServiceTest.cs

```diff
@@ -22,15 +22,33 @@ public RateServiceTests() : base("beta_rate_service")
         [Fact]
         [CrudOperations.Read]
         [Testing.Function]
-        public async Task TestRetrieve()
+        public async Task TestRetrieveStatelessRates()
         {
-            UseVCR("retrieve");
+            UseVCR("retrieve_stateless_rates");
 
             Dictionary<string, object> shipmentData = Fixtures.BasicShipment;
 
             List<StatelessRate> rates = await Client.Beta.Rate.RetrieveStatelessRates(shipmentData);
 
-            Assert.IsType<List<StatelessRate>>(rates);
+            Assert.NotNull(rates);
+            Assert.NotEmpty(rates);
```

Comment:
Question: How are these not the same...

Reply:
"Don't not exist" and "Specifically, be a list with at least one item in it".